app/controllers/schools/progress_controller.rb

```diff
@@ -55,6 +64,17 @@ def show_storage_heater_notes(school, fuel_type)
       fuel_type == :electricity && school.has_storage_heaters?
     end
 
+    def partial_consumption_data?(progress)
+      #this could be more sophisticated, relies on ordering of months array reflecting
+      #actual starting month of the target reporting period
+      first_month = progress.months.first
+      progress.monthly_usage_kwh[first_month].nil? || progress.partial_months[first_month] == true
```

Comment:
Probably don't need " == true" here? (and makes it harder to parse the evaluation, with " || " earlier in the line..?)

Reply:
Yep, have changed